pyproject.toml

```diff
@@ -60,7 +60,9 @@ dependencies = [
     # Motion Planning library
     # * 1.7.0: First version distributed on PyPI (no pre-compiled binaries for Windows OS).
     #          Fix OMPL DLL search directory.
-    "ompl>=1.7.0; platform_system != 'Windows'"
+    "ompl>=1.7.0; platform_system != 'Windows'",
+    # To capture MuJoCo warnings written to stderr
+    "wurlitzer; platform_system != 'Windows'",
```

Comment:
No need for macOS. It is already fine on MacOS thanks to my decorator `redirect_libc_stdout`.

Reply:
done